bin/v2/add-image-policies.sh

```diff
@@ -6,8 +6,11 @@ PRODUCT=$2
 COMPONENT=$3
 REGISTRY=$4
 ACR=${REGISTRY:-hmctspublic}
-COMPONENT_DIR="apps/${NAMESPACE}/${PRODUCT}-${COMPONENT}"
+APPS_DIR="../../apps/"
+COMPONENT_DIR="${APPS_DIR}/${NAMESPACE}/${PRODUCT}-${COMPONENT}"
 
+cd "$(dirname "$0")"
+pwd
```

Comment:
probably don't need to log this? assume it was just debug

Reply:
yeah it was i've removed it 👍🏼